src/components/screens/monitor/delegates/forgingDetails.css

```diff
@@ -0,0 +1,46 @@
+@import '../../../../app/variables.css';
+@import '../../../../app/mixins.css';
+
+.contentWrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+
+  & div {
+    flex-grow: 1;
+    flex-basis: 33%;
+  }
```

Comment:
💡 I think this should be replaced with http://flexboxgrid.com/ so as not to add more work to https://github.com/LiskHQ/lisk-desktop/issues/2630

using flexboxgrid will also ensure there is padding between the columns in cases like this:
![Screenshot 2019-11-14 at 16 48 16](https://user-images.githubusercontent.com/1254342/68872603-9802c680-06fe-11ea-8d4c-a6f0a10b2001.png)

Reply:
Good idea! Replaced with `flexboxgrid`.